### PR TITLE
Move to New Material Bootstrap

### PR DESCRIPTION
**Pull request by mbillow**
screen shot 2017-03-22 at 11 23 00

I moved everything over to the new bootstrap theme used on most of our other internal sites.

### CODE REVIEW

**Comment by JDrit** [comment]
When this gets deployed, all of the css / js will have to be recompiled

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -64,14 +84,13 @@
...
                 <%= yield %>
             </div><!-- /.col-md-12 -->
-        </div><!-- /.row -->
     </div><!-- /.container -->
     <br />
     <br />
     <div style="text-align:center;">
         <span class="text-center">
             <strong>
-                JD - <a href="https://github.com/JDrit/Profiles">Profiles</a>
```
Comment (liam-middlebrook) [inline app/views/layouts/application.html.erb:74]: :crying_cat_face:

Comment (JDrit) [reply app/views/layouts/application.html.erb:74]: oh you should point this to the ComputerScienceHouse repo

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -20,39 +20,59 @@
...
+            		</ul>
                 </div><!-- /.nav-collapse -->
             </div><!-- /.container -->
         </div><!-- /.navbar -->
     </navbar>
 
     <div class="container">
-        <div class="row">
             <div class="col-md-12">
-                <div class="panel profiles-panel panel-default profiles-panel-default">
```
Comment (JDrit) [inline app/views/layouts/application.html.erb:39]: why was this removed.
